packages/@aws-cdk/pipelines/lib/pipeline.ts

```diff
@@ -197,7 +197,7 @@ export class CdkPipeline extends CoreConstruct {
         stageName: 'UpdatePipeline',
         actions: [new UpdatePipelineAction(this, 'UpdatePipeline', {
           cloudAssemblyInput: this._cloudAssemblyArtifact,
-          pipelineStackName: pipelineStack.stackName,
+          pipelineStackName: pipelineStack.node.path,
```

Comment:
Property name is now wrong.

Can you `@deprecate` this one, and a new one with the correct name?